fire-cracker/SendIT-react
# #163166450 Setup environment for react/redux

> #### What does this PR do?
> * Setup environment for react/redux
Integrate code climate into the codebase
> 
> #### Description of Task to be completed?
> * Setup Webpack
> * Setup ESLint for JS and React, using Airbnb style guide
> * Setup Travis
> * Integrate code climate with Travis to run test coverage
> * Add configuration to `travis.yml` file
> * Add badges to ReadMe
> 
> #### How should this be manually tested?
> * Open the repository on GitHub
> * clone branch using `https://github.com/oyedejipeace/SendIT-react.git`
> * Check out the README to see the build badge
> * Click on it to access test details

> 
> #### What are the relevant pivotal tracker stories?
> [Setup environment for react/redux](https://www.pivotaltracker.com/story/show/163166450)
